py4DSTEM/visualize/show.py

```diff
@@ -936,11 +936,8 @@ def show_rectangles(ar,lims=(0,1,0,1),color='r',fill=True,alpha=0.25,linewidth=2
 
 def show_circles(
     ar,
-    center, 
+    center,
     R,
-    vmin = 0, 
```

Comment:
I had originally added these changes since the new default options for `show` return results that are a little confusing for `get_probe_size` (examples below with old and updated defaults). I'm realizing this isn't the best place for this change. Instead I added a plotting option with different defaults for `datacube_fns.` What do you think?
<img width="673" alt="Screen Shot 2022-10-05 at 11 44 47 AM" src="https://user-images.githubusercontent.com/55254539/194169123-06923c6d-e4d1-4ef5-94e9-484e6facd411.png">
<img width="673" alt="Screen Shot 2022-10-05 at 12 00 05 PM" src="https://user-images.githubusercontent.com/55254539/194169140-38b467b8-356b-4de8-9dd9-50bac5480a80.png">